clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -4381,6 +3979,101 @@ public boolean hasAllPartitionReportedCompleted() {
     return true;
   }
 
+  public StorageUtilizationManager getStorageUtilizationManager() {
+    return storageUtilizationManager;
+  }
+
+  public AbstractStoreBufferService getStoreBufferService() {
+    return storeBufferService;
+  }
```

Comment:
Do we need a getter for the whole `SBS`? Or would it be better to expose just a `putConsumerRecord` function? (as that seems like the only way we're using the gotten `SBS`, but maybe I missed something?)